pkg/apis/triggers/v1beta1/version_validation_test.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2021 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	"context"
+	"testing"
+
+	"github.com/tektoncd/pipeline/pkg/apis/config"
```

Comment:
Sanity check- should this be `github.com/tektoncd/triggers/pkg/apis/config`?

Reply:
good catch -- we need to add some missing helpers...most of these are being added in https://github.com/tektoncd/triggers/pull/1182/ . I'll rebase this PR once that is merged.